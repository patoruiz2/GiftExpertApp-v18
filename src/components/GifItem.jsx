const GifItem = ({ title, url }) => {
  return (
    <div className="shadow-md bg-white border-violet-400 w-fit rounded my-auto">
      <img className="object-contain" src={url} alt={title} />
      <p className="p-2">{title}</p>
    </div>
  );
};

export default GifItem;
