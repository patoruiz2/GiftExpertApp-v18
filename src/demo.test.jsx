import { render } from '@testing-library/react';
import GifItem from './components/GifItem.jsx';
import { expect } from 'vitest';

describe('Demo', () => {
  const title = "Titulo";
  const url = "someUrl";
  test('should match snapshot', () => {
    const { container } = render(<GifItem title={title} url={url} />)
    expect(container).toMatchSnapshot();
  })
})