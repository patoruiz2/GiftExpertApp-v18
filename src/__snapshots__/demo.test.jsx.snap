// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Demo > should match snapshot 1`] = `
<div>
  <div
    class="shadow-md bg-white border-violet-400 w-fit rounded my-auto"
  >
    <img
      alt="Titulo"
      class="object-contain"
      src="someUrl"
    />
    <p
      class="p-2"
    >
      Titulo
    </p>
  </div>
</div>
`;
